coderd/workspaces.go

```diff
@@ -590,7 +590,9 @@ func (api *API) postWorkspacesByOrganization(rw http.ResponseWriter, r *http.Req
 			ProvisionerJob: *provisionerJob,
 			QueuePosition:  0,
 		},
-		member.Username,
+		database.User{
+			Username: member.Username,
+		},
```

Comment:
We generally want to avoid initializing a `database.*` struct with potentially zero-valued fields. I made a change here to fetch the desired workspace owner from the database by the provided organization member parameter, and passed that in instead.